db/schema.rb

```diff
@@ -10,11 +10,56 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180611145053) do
+ActiveRecord::Schema.define(version: 20180612141601) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "active_admin_comments", force: :cascade do |t|
```

Comment:
this is the result of the migration you ran for active admin, to reset the schema correctly you can run
`rake db:drop db:create db:migrate`
You will lose all local information you created, though. That's why it's helpful to have entities created in the `seeds.rb` file